### ISSUE TITLE
You must specify a template name (-t) or path (-p). error for config

### ISSUE THREAD

**Issue by Alexey-Matjuk** [SwiftGen/SwiftGen#470]
Following command works for me:
```sh
$ swiftgen/bin/swiftgen strings -t flat-swift4 Sources/Resources/en.lproj/Localizable.strings --output Generated.swift
```
But this one not:
```sh
$ swiftgen/bin/swiftgen
Linting swiftgen.yml
You must specify a template name (-t) or path (-p).

To list all the available named templates, use 'swiftgen templates list'.
```
As i understand this error should appear only when you don't use config file, but I see it even with a config. Please help.

My config is:
```
strings:
  paths: Sources/Resources/en.lproj/Localizable.strings
  outputs:
    - templateName: flat-swift4
      output: L10n-Constants.swift
```
I'm using [5.3.0](https://github.com/SwiftGen/SwiftGen/releases/tag/5.3.0) version of SwiftGen

**Comment by AliSoftware** [comment]
Hi

How did you name your config file exactly? Are you sure you named it correctly? And where did you place it?

What does `swiftgen config lint` tells you?

If SwiftGen can't find your config file that would explain your error. Maybe you should explicitly specify its relative path when invoking SwiftGen if you're not naming or placing the file in the expected location 😉 (`swiftgen config run --config the/path` iirc)

**Comment by Alexey-Matjuk** [comment]
Hi
Actually it was a known issue. Originally it was [similar to 431](https://github.com/SwiftGen/SwiftGen/issues/431) about missing `strings.paths`. So I just blindly renamed `inputs` to `paths` without realizing why it appeared. Then I started receiving the new error about missing templates, which I wasn't able to find how to fix until I finally realized that I was looking at newer documantation version.
